Floofbot/Modules/Utilities.cs

```diff
@@ -99,5 +104,41 @@ public async Task RepeatMessage([Remainder] string message =null)
                 await Context.Channel.SendMessageAsync("Usage: `.say [message]`");
             }
         }
+
+        [Command("serverinfo")]
+        [Summary("Returns information about the current server")]
+        public async Task ServerInfo()
+        {
+            SocketGuild guild = Context.Guild;
+            int numberTextChannels = guild.TextChannels.Count;
+            int numberVoiceChannels = guild.VoiceChannels.Count;
+            string createdAt = $"Created {guild.CreatedAt.DateTime.ToShortDateString()}. " +
+                               $"That's over {Context.Message.CreatedAt.Subtract(guild.CreatedAt).Days} days ago!";
+            int totalMembers = guild.MemberCount;
+            int onlineUsers = guild.Users.Where(mem => mem.Status == UserStatus.Online).Count();
```

Comment:
I think you can just use .Count(lambda) https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.count?view=netcore-3.1#System_Linq_Enumerable_Count__1_System_Collections_Generic_IEnumerable___0__System_Func___0_System_Boolean__

Reply:
I dont see much of a difference in
`guild.Users.Count(mem =>  mem.Status == userStatus.Online);`
and what I have put here